cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -428,6 +446,18 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
         afterRender: function() {
             AbstractEditor.prototype.afterRender.call(this);
             this.$('.field-due-in input').val(this.model.get('due_num_weeks'));
+            this.$('#projected').hide();
+
+            if (this.getValue() && this.model.get('start')){
```

Comment:
it seems like this bit is mostly the same as above, could you pull it out into its own function so that it would by DRY?

Reply:
fixed!